roles/azure_manage_snapshot/tasks/main.yml

```diff
@@ -0,0 +1,3 @@
+- name: Create, delete or restore Snapshot
```

Comment:
Maybe it's worth adding a check that mandatory variables are defined before starting?

Reply:
I thought the idea of the argument_spec.yml was supposed to enforece this.